modules/seqexec/server/src/main/scala/seqexec/server/keywords/GDSClient.scala

```diff
@@ -21,9 +25,23 @@ import seqexec.server.SeqActionF
 /**
   * Gemini Data service client
   */
-final case class GDSClient(client: Client[IO], gdsUri: Uri)
+final case class GDSClient(base: Client[IO], gdsUri: Uri)(implicit timer: Timer[IO])
     extends Http4sClientDsl[IO] {
 
+  @SuppressWarnings(Array("org.wartremover.warts.Var"))
+  private val client = {
+    val max = 2
+    var attemptsCounter = 1
```

Comment:
Sorry about this

Reply:
It's lame that they kind of require you to side-effect here.